client/components/domains/contact-details-form-fields/custom-form-fieldsets/test/region-address-fieldsets.js

```diff
@@ -1,9 +1,7 @@
 /**
  * @jest-environment jsdom
  */
-
-import { expect } from 'chai';
-import { shallow } from 'enzyme';
+import { render, screen } from 'calypso/test-helpers/config/testing-library';
```

Comment:
Is it best practice to import from our helper lib instead of RTL all of the time? (Or should we still import directly from RTL if we don't need the root config?)

Reply:
In a pairing session with @flootr we just discussed that and we have a solution to reduce the confusion that's being brought here. Basically we discussed that it can be confusing because `render` is the modified render, but `screen` is the original one, provided from the `@testing-library/react` library. Will be filing a PR shortly.

Reply:
PR in https://github.com/Automattic/wp-calypso/pull/62948 to improve this and make it more obvious and transparent.